packages/flutter/lib/src/material/drawer.dart

```diff
@@ -232,16 +262,18 @@ class DrawerControllerState extends State<DrawerController> with SingleTickerPro
     if (_controller.isDismissed)
       return;
     if (details.velocity.pixelsPerSecond.dx.abs() >= _kMinFlingVelocity) {
-      final double visualVelocity = details.velocity.pixelsPerSecond.dx / _width;
+      double visualVelocity = details.velocity.pixelsPerSecond.dx / _width;
+      if(widget.type == DrawerAlignment.end) {
```

Comment:
Missing space after `if`

Reply:
Should be a switch too.  We basically don't use `operator==` to compare an enum to one if its values.  We use a `switch` statement with all of the enum values listed as cases so we know that we're handling them all properly.

Reply:
Fixed. Apologies, this should've been fixed in earlier commit 😐